src/docs/index.md

```diff
@@ -15,40 +15,46 @@ description: The landing page for Flutter documentation.
 {% endfor -%}
 </div>
 
-## What's new on this site
-
-**November 5, 2018**
-
-Welcome to the revamped Flutter website!
-
-We've spent the last few months redesigning the website and how its
-information is organized. We hope you can more easily find the docs
-you are looking for. Some of the changes to the website include:
-
-* Revised [front](/) page
-* Revised [showcase](/showcase) page
-* Revised [community](/community) page
-* Revised navigation in the left side bar
-* Table of contents on the right side of most pages
-
-Some of the new content includes:
-
-* Deep dive on Flutter internals,
-  [Inside Flutter](/docs/resources/inside-flutter)
-* [Technical videos](/docs/resources/videos)
-* [State management](/docs/development/data-and-backend/state-mgmt)
-* [Background Dart
-  processes](/docs/development/packages-and-plugins/background-processes)
-* [Flutter's build modes](/docs/testing/build-modes)
-{% comment %}
-* How to connect [a native debugger _and_
-  a Dart debugger to your app](/docs/testing/oem-debuggers)
-  (not yet complete)
-{% endcomment %}
-
-If you have questions or comments about the revamped site, [file an
+## What's new on the site
+
+**February 26, 2019**
+
+Flutter released [version
+1.2](https://developers.googleblog.com/2019/02/launching-flutter-12-at-mobile-world.html)
+today at Mobile World Congress (MWC) in Barcelona!
+
+In addition, here are some new docs we've released:
+
+* We've updated our [state management
+  advice](/docs/development/data-and-backend/state-mgmt/intro).
+  New pages include an
+  [introduction](/docs/development/data-and-backend/state-mgmt/intro),
+  [thinking declaratively](/docs/development/data-and-backend/state-mgmt/declarative), 
+  [ephemeral vs app state](/docs/development/data-and-backend/state-mgmt/ephemeral-vs-app),
+  [simple app state management](/docs/development/data-and-backend/state-mgmt/simple),
+  and [different state management
+  options](/docs/development/data-and-backend/state-mgmt/options).
+  Documenting state management is a tricky thing, as there is no one-size-fits-all
+  approach. We'd love your feedback on these new docs!
+* A new page on [Performance best practices](/docs/testing/best-practices).
+* Also at MWC, we announced a preview version of the new Dart DevTools
+  for profiling and debugging Dart and Flutter apps. You can find the docs on the
+  [DevTools wiki](https://flutter.github.io/devtools/).
+  In particular, check out the DevTool's [Flutter widget
+  inspector](https://flutter.github.io/devtools/inspector), and the [timeline
+  view](https://flutter.github.io/devtools/timeline) for profiling your Flutter
```

Comment:
nit: this makes it sound a little like both views are for profiling your application.
Maybe say Inspector view for debugging UI issues or something similar but more eloquent?